localstack/aws/skeleton.py

```diff
@@ -151,26 +149,52 @@ def invoke(self, context: RequestContext) -> HttpResponse:
                     operation.name,
                 )
                 raise NotImplementedError
-            handler = self.dispatch_table[operation.name]
 
-            # Call the appropriate handler
-            result = handler(context, instance) or {}
-            # Serialize result dict to an HTTPResponse and return it
-            return serializer.serialize_to_response(result, operation)
+            return self.dispatch_request(context, instance)
```

Comment:
potential nic, do you think `skeleton` could be called `service dispatcher` or `request dispatcher`, the concepts within are all screaming "service" and "dispatch", just a thought tho.

Reply:
yep agreed. Skeleton just came from the classic textbook RPC architecture (i.e., the server stub) :-) actually the Skeleton will be replaced soon-ish with the HandlerChain framework (currently underway in the asf branch).